copyright

```diff
@@ -0,0 +1,17 @@
+Format: http://dep.debian.net/deps/dep5/
+
+Files: *
+Copyright: Copyright 2021, Canonical Ltd., All Rights Reserved.
+License: GPL-3
```

Comment:
Do we really want this license? AFAIK we prefer the Apache license, e.g. [here](https://github.com/openstack/charm-keystone/blob/master/copyright).